src/main/java/alexiil/mods/load/MinecraftDisplayer.java

```diff
@@ -249,41 +245,41 @@ public String randomBackground(String currentBG) {
         if (randomBackgroundArray.length == 1){
             return randomBackgroundArray[0];
         }
-    	//System.out.println("currentBG is: "+currentBG);
+    	BetterLoadingScreen.log.debug("currentBG is: " + currentBG);
     	Random rand = new Random();
     	String res = randomBackgroundArray[rand.nextInt(randomBackgroundArray.length)];
-        //System.out.println("New res is: "+res);
-        //System.out.println("Does alreadyUsedBGs contain res?: "+String.valueOf(alreadyUsedBGs.contains(res)));
+        BetterLoadingScreen.log.trace("New res is: " + res);
+        BetterLoadingScreen.log.trace("Does alreadyUsedBGs contain res?: "+String.valueOf(alreadyUsedBGs.contains(res)));
         if (randomBackgroundArray.length == alreadyUsedBGs.size()) {
             alreadyUsedBGs.clear();
         }
     	while (res.equals(currentBG) || alreadyUsedBGs.contains(res)) {
     		res = randomBackgroundArray[rand.nextInt(randomBackgroundArray.length)];
-    		//System.out.println("Rerolled res is: "+res);
+    		BetterLoadingScreen.log.trace("Rerolled res is: " + res);
     	}
         alreadyUsedBGs.add(res);
-    	//System.out.println("res is: "+res);
+        BetterLoadingScreen.log.trace("res is: " + res);
     	return res;
     }
 
     public String randomTooltip(String currentTooltip) {
         if (randomTips.length == 1){
             return randomTips[0];
         }
-        //System.out.println("currentTooltip is: " + currentTooltip);
+        BetterLoadingScreen.log.trace("currentTooltip is: " + currentTooltip);
         Random rand = new Random();
         String res = randomTips[rand.nextInt(randomTips.length)];
-        //System.out.println("New res (tooltip) is: "+res);
-        //System.out.println("Does alreadyUsedTooltips contain res?: "+String.valueOf(alreadyUsedTooltips.contains(res)));
+        BetterLoadingScreen.log.trace("New res (tooltip) is: " + res);
+        BetterLoadingScreen.log.trace("Does alreadyUsedTooltips contain res?: " + alreadyUsedTooltips.contains(res));
         if (randomTips.length == alreadyUsedTooltips.size()) {
             alreadyUsedTooltips.clear();
         }
         while (res.equals(currentTooltip) || alreadyUsedTooltips.contains(res)) {
             res = randomTips[rand.nextInt(randomTips.length)];
-            //log.info("Rerolled res (tooltip) is: "+res);
+            BetterLoadingScreen.log.trace("Rerolled res (tooltip) is: " + res);
         }
         alreadyUsedTooltips.add(res);
-        //log.info("res is: "+res);
+        BetterLoadingScreen.log.trace("res is: " + res);
```

Comment:
Is there a reason to reenable those logs? like it won't appear in the log because it will filter them in the forge logs, but it is still writing it in the base log

Reply:
Isn't "trace" disabled by default? Hmmm, if it isn't I need to check those

Reply:
It appears something is turning trace on; I'll comment them back out